assets/dev/js/admin/admin.js

```diff
@@ -334,6 +334,8 @@ import FloatingButtonsHandler from 'elementor/modules/floating-buttons/assets/js
 
```

Comment:
Playwright test its in Pro [https://github.com/elementor/elementor-pro/pull/4356#pullrequestreview-2185964964](https://github.com/elementor/elementor-pro/pull/4356#pullrequestreview-2185964964) because WooCommerce is installed in Pro Playwright